site/content/concepts/tenancy.md

```diff
@@ -1,12 +1,25 @@
 +++
-title = "Multi Tenancy"
+title = "Multitenancy"
 weight = 405
 +++
 
-Hono has a multi tenancy concept based on the handling of tenants as own *entities*. All functionality of Hono is 
+Hono is designed to structure the set of all internally managed data and data streams into strictly isolated subsets. 
+This includes the registration data and credentials of devices, internal users that are used for authentication, 
+and the *Business Applications* that are part of such subsets as well.
+
+This way of *strict* isolation is generally known as multitenancy, where a **tenant** is the term for such a subset.
+Such an isolation is essential for enabling a scalable distributed architecture to handle independent subsets as if each subset had its
+own installation (which would be much harder to maintain and would not benefit from runtime cost sharing).
+
+This is why Hono implements a multitenancy concept.
```

Comment:
this isolated sentence sounds a little strange when reading through the section. I would propose to simply omit it. You already gave the reasons why we do this in the previous paragraph ...